### PR TITLE
ref: review `remote add/modify --local` info.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
rel https://github.com/iterative/dvc.org/issues/2970

### CODE REVIEW

File: content/docs/command-reference/remote/add.md [hunk 1]
```diff
@@ -233,12 +233,12 @@ $ dvc remote modify myremote oss_endpoint endpoint
...
 To set key id, key secret and endpoint (or any other OSS parameter), use
-`dvc remote modify`. Example usage is show below. Make sure to use the `--local`
-option to avoid committing your secrets with Git:
+`dvc remote modify` as show below. Use the `--local` option to avoid committing
+your secrets to Git:
 
 ```dvc
-$ dvc remote modify myremote --local oss_key_id 'mykey'
-$ dvc remote modify myremote --local oss_key_secret 'mysecret'
+$ dvc remote modif --localy myremote oss_key_id 'mykey'
```
Comment (shcheklein) [inline content/docs/command-reference/remote/add.md:240]: typo?

File: content/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -616,8 +621,8 @@ more information.
   [delegated] to the service account if needed.
 
   ```dvc
-  $ dvc remote modify myremote \
-                 gdrive_service_account_user_email 'myemail-addr'
+  $ dvc remote modify --local myremote \
```
Comment (shcheklein) [inline content/docs/command-reference/remote/modify.md:624]: this is a very good example when actual changes and refactoring (changing `--local` location) could go in two different PRs. 

This is PR is supposed to be quick and easy, but requires time to review properly.

**Comment by shcheklein** [comment]
any substantial changes (we didn't have `--local` and now we set it) ... something besides moving things around?

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/remote/add.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/add.md:240]: Thanks
```suggestion
$ dvc remote modify --localy myremote oss_key_id 'mykey'
```

Comment (jorgeorpinel) [reply content/docs/command-reference/remote/add.md:240]: Hmmm for some reason I can no longer commit suggestions on Github :(

Comment (jorgeorpinel) [reply content/docs/command-reference/remote/add.md:240]: Now the option exists again! Gh bug fixed I suppose. 🤷

File: content/docs/command-reference/remote/modify.md [hunk 3]
```diff
@@ -156,10 +156,14 @@ By default, DVC authenticates using your AWS CLI
 (if set). This uses the default AWS credentials file. Use the following
 parameters to customize the authentication method:
 
+> If any values given to the parameters below contain sensitive user info, add
+> them with the `--local` option, so they're written to a Git-ignored config
+> file.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:161]: Added this note near the top of every remote section...

File: content/docs/command-reference/remote/modify.md [hunk 4]
```diff
@@ -228,10 +232,6 @@ parameters to customize the authentication method:
   $ dvc remote modify myremote ssl_verify path/to/ca_bundle.pem
   ```
 
-> The credentials file path, access key and secret, and other options contains
-> sensitive user info. Therefore, it's safer to add it with the `--local`
-> option, so it's written to a Git-ignored config file.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:233]: ... and removed the existing ones which were all different and near the bottom.

File: content/docs/command-reference/remote/modify.md [hunk 5]
```diff
@@ -156,10 +156,14 @@ By default, DVC authenticates using your AWS CLI
...
 
+> If any values given to the parameters below contain sensitive user info, add
+> them with the `--local` option, so they're written to a Git-ignored config
+> file.
+
 - `profile` - credentials profile name to access S3:
 
   ```dvc
-  $ dvc remote modify myremote profile myprofile
+  $ dvc remote modify --local myremote profile myprofile
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:166]: Added --local here...

File: content/docs/command-reference/remote/modify.md [hunk 6]
```diff
@@ -553,22 +558,22 @@ for a full guide on using Google Drive as DVC remote storage.
...
   ```
 
 - `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
   access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
 
   ```dvc
-  $ dvc remote modify myremote \
+  $ dvc remote modify --local myremote \
         gdrive_user_credentials_file .dvc/tmp/mycredentials.json
   ```
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:578]: ...and in all these

File: content/docs/command-reference/remote/modify.md [hunk 7]
```diff
@@ -616,8 +621,8 @@ more information.
   [delegated] to the service account if needed.
 
   ```dvc
-  $ dvc remote modify myremote \
-                 gdrive_service_account_user_email 'myemail-addr'
+  $ dvc remote modify --local myremote \
+                      gdrive_service_account_user_email 'myemail-addr'
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:625]: ...and here.

File: content/docs/command-reference/remote/modify.md [hunk 8]
```diff
@@ -879,7 +889,8 @@ by HDFS. Read more about by expanding the WebHDFS section in
   `false` this setting is ignored.
 
   ```dvc
-  $ dvc remote modify myremote kerberos_principal some_principal_name
+  $ dvc remote modify --local myremote \
+                              kerberos_principal some_principal_name
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:893]: Added --local here as well...

File: content/docs/command-reference/remote/modify.md [hunk 9]
```diff
@@ -890,7 +901,7 @@ by HDFS. Read more about by expanding the WebHDFS section in
   kerberos must be enabled for this to work.
 
   ```dvc
-  $ dvc remote modify myremote proxy_to some_proxy_user
+  $ dvc remote modify --local myremote proxy_to some_proxy_user
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:904]: ...and here

File: content/docs/command-reference/remote/modify.md [hunk 10]
```diff
@@ -910,7 +921,8 @@ by HDFS. Read more about by expanding the WebHDFS section in
   used.
 
   ```dvc
-  $ dvc remote modify myremote token SOME_BASE64_ENCODED_TOKEN
+  $ dvc remote modify --local myremote \
+                              token SOME_BASE64_ENCODED_TOKEN
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:925]: ...and here.

**Review by jorgeorpinel — COMMENTED** [review]
> any substantial changes 

Yes @shcheklein :

File: content/docs/command-reference/remote/modify.md (hunk 6 above)
Comment (shcheklein) [reply content/docs/command-reference/remote/modify.md:578]: this is not sensitive and could be shared in some cases (to use multiple GDrive remotes for example)

File: content/docs/command-reference/remote/modify.md (hunk 7 above)
Comment (shcheklein) [reply content/docs/command-reference/remote/modify.md:625]: I would not make this one sensitive as well

File: content/docs/command-reference/remote/modify.md (hunk 8 above)
Comment (shcheklein) [reply content/docs/command-reference/remote/modify.md:893]: also not sure about this

File: content/docs/command-reference/remote/modify.md (hunk 9 above)
Comment (shcheklein) [reply content/docs/command-reference/remote/modify.md:904]: and here - all look like per project configs

File: content/docs/command-reference/remote/modify.md (hunk 6 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md]: OK. Reverted these 3.

File: content/docs/command-reference/remote/modify.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md:625]: Reverted.

File: content/docs/command-reference/remote/modify.md (hunk 8 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md:893]: Not sure either. Reverted.

File: content/docs/command-reference/remote/modify.md (hunk 9 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md]: Reverted.
